conf/base/bblayers.conf

```diff
@@ -3,22 +3,24 @@
 POKY_BBLAYERS_CONF_VERSION = "2"
 
 BBPATH = "${TOPDIR}"
+BSPDIR := "${@os.path.abspath(os.path.dirname(d.getVar('FILE', True)) + '/../..')}"
+
 BBFILES ?= ""
 
 BBLAYERS ?= " \
-  ##OEROOT##/meta \
-  ##OEROOT##/meta-poky \
-  ##OEROOT##/meta-yocto-bsp \
-  ##OEROOT##/meta-qt4 \
-  ##OEROOT##/meta-qt5 \
-  ##OEROOT##/meta-uav \
-  ##OEROOT##/meta-ros \
-  ##OEROOT##/meta-intel-aero \
-  ##OEROOT##/meta-intel-aero-connectivity \
-  ##OEROOT##/meta-intel-realsense \
-  ##OEROOT##/intel-aero-samples \
-  ##OEROOT##/meta-openembedded/meta-oe \
-  ##OEROOT##/meta-openembedded/meta-python \
-  ##OEROOT##/meta-openembedded/meta-networking \
-  "
+  ${BSPDIR}/sources/poky/meta \
```

Comment:
do we need to have the extra `sources` directory instead of keeping it on poky/?  I think keeping it on poky would be more compatible with the yocto manual and also avoid the changes to manifest to checkout source in a different dir from what's there.